states/mainmenu.py

```diff
@@ -67,7 +67,7 @@ def start(self) -> None:
             self.save()
         except SystemExit:
             # Save and exit immediately.
-            if not self.model.is_player_dead:
+            if not self.model.is_player_dead():
```

Comment:
Oh no, how long has that been there?  I should have made it a property in the first place.